docs/what_is_data_science/How you can learn Data Science.md

```diff
@@ -12,3 +12,9 @@ You can also learn lot about data science by simply getting to know the wider cr
 - [AnalystX](https://analystx.uk/)
 
 These have some overlap, but each has it's own niche, depending on what language or area you want to focus more on (e.g. Python, R, specific use cases).
+
+## NHS England Data Science Professional Development Newsletter
+The NHS England Data Science Professional Development Functional Team puts together a monthly newsletter with valuable insights, training opportunities and events for people interested in learning more about various aspects of data science, further developing their skills, and progressing in their career. As this is targeted towards members of the NHS England Data Science team some links may only be accessible to those with the necessary login credentials, however the newsletter and its archive are available for all at the link below.
```

Comment:
I think this could be made a bit more brief  - people outside the business might not be so interested in the internal divisions of our team (e.g. "NHS England Data Science Professional Development Function Team") - instead, it could just say "We make a monthly newsletter..."